src/components/app-configure/CreateApp.vue

```diff
@@ -13,52 +13,62 @@ import { useToast } from '@/components/lib/VToast'
 import { createApp } from '@/services/gateway.service'
 import { useAppsStore } from '@/stores/apps.store'
 import { useChainManagementStore } from '@/stores/chainManagement.store'
-import { useLoaderStore } from '@/stores/loader.store'
 import { RegionMapping, regions } from '@/utils/constants'
 import { createAppConfig } from '@/utils/createAppConfig'
 import { createTransactionSigner } from '@/utils/signerUtils'
+import validateRPCandChainId from '@/utils/validateRPCandChainId'
 
 const router = useRouter()
-const loaderStore = useLoaderStore()
 const toast = useToast()
 const appsStore = useAppsStore()
 const appName = ref('')
 const selectedChainId: Ref<number | null> = ref(null)
 const hasAppNameError = ref(false)
 const selectedRegion = ref(regions[0])
 const chainManagementStore = useChainManagementStore()
+const showLoader = ref(false)
 
 const emit = defineEmits(['close'])
 
+function getPayloadForCreateApp() {
+  return {
+    name: appName.value,
+    chain: selectedChainId.value,
+    default_chain: selectedChainId.value,
+    region: RegionMapping[selectedRegion.value.value],
```

Comment:
Remove region, I don't think we need region, check with the backend team. Since we are not using store anymore, I don't think this is required

Reply:
[done](https://github.com/arcana-network/developer-dashboard/pull/249/commits/abe957e7ecec9beaf3a86e05e46e062af0c3ac36).